### PR TITLE
docs: helm install with namespace

### PR DESCRIPTION
**Pull request by haoxins**

### CODE REVIEW

**Comment by tao12345666333** [comment]
Thanks for your contribution. 

I saw you opened #203, #204, #205 at the same time.
I don't think this is necessary. You can cherry-pick these commits to a PR.

**Comment by haoxins** [comment]
I just submitted the PRs from the online editor.
but fine.
cherry picked.

File: docs/en/latest/apisix-ingress-controller.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ To install apisix-ingress-controller which release name is `apisix-ingress-contr
 ```bash
 helm repo add apisix https://charts.apiseven.com
 helm repo update
-helm install apisix-ingress-controller apisix/apisix-ingress-controller --namespace ingress-apisix
+helm install apisix-ingress-controller apisix/apisix-ingress-controller --namespace ingress-apisix --create-namespace
```
Comment (zaunist) [inline docs/en/latest/apisix-ingress-controller.md:37]: ref https://github.com/apache/apisix-helm-chart/issues/176, we need add ` --set ingress-controller.config.apisix.serviceNamespace=<your namespace>` flag.

Comment (haoxins) [reply docs/en/latest/apisix-ingress-controller.md:37]: is this another `topic`? the `--create-namespace` only fixes the `namespace` not exists issue, not others.

Comment (zaunist) [reply docs/en/latest/apisix-ingress-controller.md:37]: Got it.

**Review by zaunist — APPROVED** [review]

**Review by tao12345666333 — APPROVED** [review]

**Review by tao12345666333 — APPROVED** [review]

**Review by tokers — APPROVED** [review]

**Pull request merged**